src/badge/badge.js

```diff
@@ -1,20 +1,31 @@
 import React from 'react';
-import { Text, View, StyleSheet } from 'react-native';
+import { Text, View, StyleSheet, TouchableOpacity } from 'react-native';
 
 let styles = {};
```

Comment:
I think we moved away from this. @binoy14 I remember you refactored this as part of the props PR right?